### ISSUE TITLE
Being able to Bold multiple List Blocks in editor 

### ISSUE THREAD

**Issue by xecutable** [WordPress/gutenberg#58491]
Hi,

This has been bugging me forever, but when I create a few List blocks like the following:

1.Example1
2.Example2
3.Example3
4.Example4

I'd like to be able to highlight them with the mouse, press CTRL + B and bold all three of them. At the moment and has been like this for quite some time, you can only bold them individually, or go to the block options and select a style there, which takes more time to do than to actually bold each individual line.

Any chance we can get a typography styling bar while selecting multiple blocks of an identical style like Lists, as for now we only get a group up bar and that's it. The text styling has to be done on each one individually, unless you want to get in the settings.

Can't see why this can't be available on the fly while working and speed up the process significantly especially on long lists.

This is what I'm talking about visually:

![how-it-is](https://github.com/WordPress/gutenberg/assets/47508335/d80bd0c1-516e-470a-ad47-dd30c1c2317f)
![how-it-is2](https://github.com/WordPress/gutenberg/assets/47508335/b1e717ec-5de7-440a-82e9-f41bb5975d2c)

Basically get the typography bar from the second picture, to appear on the top when highlighting multiple lists.

**Comment by Mamaduka** [comment]
Thanks for creating the issue, @xecutable!

This looks similar to the previous request https://github.com/WordPress/gutenberg/issues/53197#issuecomment-1679454943, which was closed as a duplicate of #27221.

Feel free to add a comment to the main issue.

**Comment by xecutable** [comment]
Yup I see it now. I can't find the status on this though. Is it in progress, was it reject or what's going on?

**Comment by Mamaduka** [comment]
@xecutable, the main issue is to open; I believe there are some blockers that need to be resolved first.

Feel free to comment on #27221 and check the status.

**Comment by xecutable** [comment]
> @xecutable, the main issue is to open; I believe there are some blockers that need to be resolved first.
> 
> Feel free to comment on #27221 and check the status.

I see, thank you for the assistance. I'll continue to follow the thread you've linked.
